python/ray/tune/suggest/ax.py

```diff
@@ -124,8 +124,14 @@ def __init__(self,
         assert ax is not None, """Ax must be installed!
             You can install AxSearch with the command:
             `pip install ax-platform sqlalchemy`."""
+        
+        ax_mode_is_min = ax.experiment.optimization_config.objective.minimize 
+        expected_mode = "min" if ax_mode_is_min else "max"
```

Comment:
This doesn't work on latest Ax - did you mean to use `self._ax.experiment....`?

Reply:
Oy yes, sorry about that!